osp/serializers/fields.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework import serializers
+
+from osp.models import (
+    Question, Choice, Checkbox, Dropdown,
+    Paragraph, ShortAnswer, Date, Time, FileUpload
+)
+from osp.serializers.question import QuestionSerializer
+
+class ChoiceSerializer(serializers.ModelSerializer):
```

Comment:
I don't see some explicit logic for creation of options?
Is it working properly without explicit logic?